types/macaca-circular-json/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```

Comment:
Why isn't `strictNullChecks` on?

Reply:
It's same with https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/circular-json/tsconfig.json#L9, so I didn't change it.

Reply:
Everything looks good now except this. If you flip it on I can pull in this change.